src/isomorphic/React.js

```diff
@@ -28,6 +28,13 @@ var createFactory = ReactElement.createFactory;
 var cloneElement = ReactElement.cloneElement;
 
 if (__DEV__) {
+  console.log(
```

Comment:
I also couldn't decide if it was exactly a warning. It's more along the lines of "Listening on http://localhost:3000", which everyone is ok with.